src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -58,14 +98,6 @@ const BioAndDMSetting: React.FC<BioAndDMSettingProps> = ({
     }
   };
 
-  const handleClickTextArea = () => {
-    ToastMessage.show({
-      type: 'asNative',
-      text1: 'You cannot send yourself messages.',
-      position: 'bottom'
-    });
-  };
-
   const ref = React.useRef(true);
 
   React.useEffect(() => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The old hunk had a function `handleClickTextArea` which was used to show a toast message when the text area was clicked. This function has been removed in the new hunk. If this functionality is no longer needed, then it's fine. Otherwise, it should be added back.



<!-- This is an auto-generated comment by OSS CodeRabbit -->